common-content/en/module/tools/binary-and-hexidecimal-numbers/index.md

```diff
@@ -5,7 +5,7 @@ time = 90
 facilitation = false
 emoji= "📖"
 [objectives]
-    1="Convert numbers between any two of: binary, decimal, hexidecimal"
+    1="Convert signed and unsigned numbers between any two of: binary, decimal, hexidecimal"
```

Comment:
```suggestion
    1="Convert signed and unsigned numbers between any two of: binary, decimal, hexadecimal"
```

Psst, throughout, you are consistently mis-spelling hexadecimal as hexidecimal. It's got an a in it.